goship.go

```diff
@@ -321,6 +331,7 @@ func DeployOutputHandler(w http.ResponseWriter, r *http.Request, env string, for
 }
 
 func DeployLogHandler(w http.ResponseWriter, r *http.Request, full_env string, environment goship.Environment) {
+	u, _ := getUser(r)
```

Comment:
> `u, _ := ...`

Throwing away an error here - if it's non-fatal, should it be logged? Perhaps it depends on the kind of error?